source/fe/fe_data.cc

```diff
@@ -142,6 +142,7 @@ FiniteElementData<dim>::FiniteElementData(
   const Conformity                      conformity,
   const BlockIndices &                  block_indices)
   : reference_cell_kind(reference_cell)
+  , data(data)
```

Comment:
From reading how the argument `data` is used below, the `data` member variable is duplicative of information we already store. I'm not in favor of that.